### DISCUSSION TITLE
Is it possible to create a new release containing AzIdentity v1.0 in the nearest future?

### DISCUSSION THREAD

**Discussion by tinolazreg** [microsoft/go-mssqldb#41] [Q&A] [answered]
We are unable to use this package, since we are dependent on AzIdentity V1.0. I just noticed that https://github.com/microsoft/go-mssqldb/commit/c33ed631655f2055a27eb1d2a74f14eae81ad946 was merged in a couple of week ago. It would be greatly appreciated if it could be included in a release in the nearest future :)

**Comment by shueybubbles** [comment, marked as answer]
I will push a new tag as soon as I fix #39 
In the meantime you could bump to `@latest`.  The version tags are just friendly names for a commit.

**Reply by tinolazreg** [reply]
Thank you! :)
